package.json

```diff
@@ -66,6 +66,7 @@
     "regenerator-runtime": "0.13.7",
     "universal-cookie": "4.0.4",
     "url": "0.11.0",
+    "uuid": "^9.0.0",
```

Comment:
nit: let's keep this dependency pinned like the other dependencies in this file (i.e., remove the `^`)